plugin/ref.vim

```diff
@@ -20,7 +20,11 @@ vnoremap <silent> <Plug>(ref-keyword) :<C-u>call ref#K('visual')<CR>
 
 if !exists('g:ref_no_default_key_mappings') || !g:ref_no_default_key_mappings
   silent! nmap <silent> <unique> K <Plug>(ref-keyword)
-  silent! vmap <silent> <unique> K <Plug>(ref-keyword)
+  if exists(':xmap') == 2
```

Comment:
I think this check is unnecessary.  Just use `:xmap` instead of `:vmap`.

Reply:
Okay, refine it.

Reply:
If can fix together `vnoremap` Plug in mapping (to `xnoremap`).

Do you require or not?

Reply:
Fix together.